### PR TITLE
Added `LibraryCharges` for monatomic ions

### PR DESCRIPTION
**Pull request by dotsdl**
This addresses #5.

I have added the monatomic ion `LibraryCharges` added in openforcefield/openforcefield#563. I have also added appropriate entries under *Versions* in the README, and attempted to give a tree-like structure showing our version history.

### CODE REVIEW

**Comment by dotsdl** [comment]
This is ready for review. Please let me know if anything additional is needed to proceed to merge. Thanks!

File: README.md [hunk 1]
```diff
@@ -73,7 +73,13 @@ Force fields moving forward will be called `name-X.Y.Z`
 
 ## Versions
 
-`v1.1.0 Parsley `: This minor release contains following changes. (1) Addition of new proper torsions and improper torsions for tetrazole; (2) Corrections to N-N bond rotation periodicity; (3) Removal of redundant periodicity component in `t19`; (4) Addition of three new bond and angle terms, `a22a`, `b14a` and `b36a`.
+- `v1.0.0 Parsley` : First major forcevield release.
```
Comment (davidlmobley) [inline README.md:76]: Typo, "forcefield" not "forcevield" ;)

Comment (dotsdl) [reply README.md:76]: Ah, thanks for the catch! I'll fix this in the new, separate PRs.

**Comment by dotsdl** [comment]
@j-wags done! See #9 and #10. I will close this PR.

**Pull request closed without merging**